root/components/common-macros.tt

```diff
@@ -350,8 +350,12 @@ END -%]
     IF namevar AND action == 'show' AND content != ''
             AND (noescape ? html_unescape(content.remove('</?span\b[^>]*>')) : content) != entity.name;
         options.unshift('name_variation');
-        hover = hover != '' ? l('{name} – {additional_info}', { name => entity.name, additional_info => hover })
-                            : entity.name;
+        IF (entity.sort_name && entity.sort_name == entity.name);
```

Comment:
We also use `namevar` when recordings are linked to using a track name. So I think it's correct to do the change in `link_artist` instead, because artists are the only entities with sort names (well, non-alias ones).